orchagent/portsorch.cpp

```diff
@@ -110,105 +124,88 @@ PortsOrch::PortsOrch(DBConnector *db, vector<string> tableNames) :
         m_portListLaneMap[tmp_lane_set] = port_list[i];
     }
 
-    // Get default bridge id and VLAN id
-    vector<sai_attribute_t> switch_attrs;
-    sai_attribute_t switch_attr;
-    switch_attr.id = SAI_SWITCH_ATTR_DEFAULT_1Q_BRIDGE_ID;
-    switch_attrs.push_back(switch_attr);
-    switch_attr.id = SAI_SWITCH_ATTR_DEFAULT_VLAN_ID;
-    switch_attrs.push_back(switch_attr);
+    /* Get default 1Q bridge and default VLAN */
+    vector<sai_attribute_t> attrs;
+    attr.id = SAI_SWITCH_ATTR_DEFAULT_1Q_BRIDGE_ID;
+    attrs.push_back(attr);
+    attr.id = SAI_SWITCH_ATTR_DEFAULT_VLAN_ID;
+    attrs.push_back(attr);
 
-    status = sai_switch_api->get_switch_attribute(gSwitchId, switch_attrs.size(), switch_attrs.data());
+    status = sai_switch_api->get_switch_attribute(gSwitchId, attrs.size(), attrs.data());
     if (status != SAI_STATUS_SUCCESS)
     {
-        SWSS_LOG_ERROR("Failed to get default 1Q_BRIDGE: %d", status);
+        SWSS_LOG_ERROR("Failed to get default 1Q bridge and/or default VLAN, rv:%d", status);
         throw "PortsOrch initialization failure";
     }
-    m_default1QBridge = switch_attrs[0].value.oid;
-    m_defaultVlan = switch_attrs[1].value.oid;
 
-    // Get all bridge ports in the default 1Q bridge
-    sai_attribute_t bridge_attr;
-    bridge_attr.id = SAI_BRIDGE_ATTR_PORT_LIST;
-    vector<sai_object_id_t> bp_list(100);
-    bridge_attr.value.objlist.list = bp_list.data();
-    bridge_attr.value.objlist.count = bp_list.size();
+    m_default1QBridge = Port("1QBridge", Port::DOT1Q_BRIDGE);
+    m_default1QBridge.m_bridge_id = attrs[0].value.oid;
+    m_portList[m_default1QBridge.m_alias] = m_default1QBridge;
```

Comment:
i will remove them.

---
In reply to: [126867783](https://github.com/Azure/sonic-swss/pull/250#discussion_r126867783) [](ancestors = 126867783)